src/renderer/views/UserPlaylists/UserPlaylists.js

```diff
@@ -242,6 +247,10 @@ export default defineComponent({
       })
     },
 
+    keyboardShortcutHandler: function (event) {
+      return ctrlFHandler(event, this.$refs.searchBar)
```

Comment:
I don't think the return here is needed (ditto for all the other ctrlFHandler usages).

Reply:
Just tested and can confirm. I'll remove those if we think that is confusing.